doc/source/release/1.19.0-notes.rst

```diff
@@ -24,14 +24,14 @@ Calling ``np.array([[1, [1, 2, 3]])`` will issue a ``DeprecationWarning`` as
 per `NEP 34`_. Users should explicitly use ``dtype=object`` to avoid the
 warning.
 
-.. _`NEP 34`: https://numpy.org/neps/nep-0034.html
+.. _`NEP 34`: https://numpy.org/neps/nep-0034-infer-dtype-is-object.html
```

Comment:
Now that #16450 is merged, the can be done via intersphinx rather than explicitly creating a link here.

Reply:
Are you sure? Check that is in `https://numpy.org/neps/object.inv` to `conf.py`

Reply:
I see `nep34` as a valid target in the `objects.inv` file:

```
python -msphinx.ext.intersphinx https://numpy.org/neps/objects.inv
```

Good point though - it seems the neps are missing from the intersphinx mapping in the main `conf.py` for the NumPy docs. Adding it now...

Reply:
Deferring NEP intersphinx enhancement for another PR.  This PR is for broken links.